testing/tests/DevExpress.ui.widgets/popup.tests.js

```diff
@@ -2198,6 +2375,20 @@ QUnit.module('rendering', {
 
         assert.ok($('.' + POPUP_BOTTOM_CLASS).dxToolbar('instance').option('compactMode'), 'bottom toolbar has the compact option');
     });
+
+    QUnit.test('dx-popup-content-scrollable class should be attached when preventScrollEvents is used', function(assert) {
+        this.instance.show();
+
+        const $popupContent = this.instance.$content();
+
+        assert.strictEqual($popupContent.hasClass('dx-popup-content-scrollable'), true, 'scrollable class is attached');
```

Comment:
Shouldn't we move `dx-popup-content-scrollable` to a constant?

Reply:
https://github.com/DevExpress/DevExtreme/pull/23926/commits/fd2b6b773237f86f0fd2b04d5786a12da38224c0